README.md

```diff
@@ -9,7 +9,63 @@ syslog-ng is an enhanced log daemon, supporting a wide range of input
 and output methods: syslog, unstructured text, message queues,
 databases (SQL and NoSQL alike) and more.
 
-Key features:
+Quickstart
+==========
+
+The easiest configuration that accepts log system logs on /dev/log (or from
+systemd) and writes everything to a single file:
+
+```
+@version: 3.8
+@include "scl.conf"
+
+log {
+	source { system(); };
+	destination { file("/var/log/syslog"); };
+};
+```
+
+This one also processes logs from the network (TCP/514 by default):
+
+```
+@version: 3.8
+@include "scl.conf"
+
+log {
+	source {
+		system();
+		network();
+	};
+	destination { file("/var/log/syslog"); };
+};
+```
+Structured/application logging, submission via JSON, output in key=value format.
+
+```
+@version: 3.8
+@include "scl.conf"
+
+log {
+	source { system(); };
+	destination { file("/var/log/app.log" template("$(format-welf --subkeys .cim.)\n")); };
```

Comment:
I see. I thought the line above the example describes what the example does
(Structured/application logging, submission via JSON), that's why I thought
it should be format-json.



On Sun, May 15, 2016 at 5:56 PM, Balazs Scheidler <notifications@github.com>
wrote:

> In README.md
> <https://github.com/balabit/syslog-ng/pull/1052#discussion_r63293797>:
>
> > +	source {
> > +		system();
> > +		network();
> > +	};
> > +	destination { file("/var/log/syslog"); };
> > +};
> > +```
> > +Structured/application logging, submission via JSON, output in key=value format.
> > +
> > +```
> > +@version: 3.8
> > +@include "scl.conf"
> > +
> > +log {
> > +	source { system(); };
> > +	destination { file("/var/log/app.log" template("$(format-welf --subkeys .cim.)\n")); };
>
> nope, I wanted format-welf just to show the flexibility. and it is
> supported by splunk natively for instance.
> … <#m_-5570373722598911680_>
> -- Bazsi
> On Sun, May 15, 2016 at 10:48 AM, Robert Fekete ***@***.***> wrote: In
> README.md <#1052 (comment)
> <https://github.com/balabit/syslog-ng/pull/1052#discussion_r63288401>>: >
> + source { > + system(); > + network(); > + }; > + destination {
> file("/var/log/syslog"); }; > +}; > +``` > +Structured/application logging,
> submission via JSON, output in key=value format. > + > +``` > ***@***.***:
> 3.8 > ***@***.*** "scl.conf" > + > +log { > + source { system(); }; > +
> destination { file("/var/log/app.log" template("$(format-welf --subkeys
> .cim.)\n")); }; Didn't you mean format-json here? — You are receiving this
> because you authored the thread. Reply to this email directly or view it on
> GitHub <
> https://github.com/balabit/syslog-ng/pull/1052/files/a693b2fa7e5b671471320adb66cd7e105194c9e2#r63288401>
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/1052/files/a693b2fa7e5b671471320adb66cd7e105194c9e2#r63293797>
>